cli/pub_command.go

```diff
@@ -119,7 +120,23 @@ func (c *pubCmd) doReq(nc *nats.Conn, progress *uiprogress.Bar) error {
 			return err
 		}
 
-		m, err := nc.RequestMsg(msg, opts.Timeout)
+		var m *nats.Msg
+		if c.replyTo == "" {
+			m, err = nc.RequestMsg(msg, opts.Timeout)
+			if err != nil {
+				return err
+			}
+		} else {
+			sub, e := nc.SubscribeSync(c.replyTo)
+			if e != nil {
+				return e
+			}
+			if e := nc.PublishMsg(msg); e != nil {
```

Comment:
split this ie. 

```
err = nc.Publish....
if err != nil { }
```

Thats consistent with the style already in place here

Reply:
fixed